### PR TITLE
WSMan Poller support for wsman discovered nodes

### PR DESCRIPTION
**Pull request by phelpdh**
- Implement wsman pollers
- Fix wsman-job where pollers were all polling the same target.
- Clean up formatting of wsman-job.js
- Fix wsman obm service functionality

### CODE REVIEW

File: lib/task-data/base-tasks/create-default-pollers.js [hunk 1]
```diff
@@ -7,8 +7,7 @@ module.exports = {
     injectableName: 'Task.Base.Pollers.CreateDefault',
     runJob: 'Job.Pollers.CreateDefault',
     requiredOptions: [
-        'pollers',
-        'nodeId'
```
Comment (michaelsteven) [inline lib/task-data/base-tasks/create-default-pollers.js]: Add a comment explaining why this is removed for this poller.

Comment (phelpdh) [inline lib/task-data/base-tasks/create-default-pollers.js]: When calling wsman-post-discovery taskgraph via workflow-tool, the node id is passed in as the first arg to the runGraph() function.  For some reason, the passed in node id is not found by the create-default-pollers task.  I've also tried passing node id in an options object, which also fails.

File: lib/jobs/dell-wsman-discovery.js [hunk 2]
```diff
@@ -660,20 +660,20 @@ function DellWsmanDiscoveryJobFactory(
...
                                                 self.addCatalog(node_, device.summary, 'DeviceSummary'),
                                                 self.addCatalog(node_, dmiCatalog, 'dmi'),
                                                 self.doPublishDiscovered(node_)])
                             .then(function(){
                                 self.createEnclosure(node_, device.ipAddress)
                                 .then(function(encNode){
-                                    return self.createRedfishObm(encNode, device.ipAddress, 1)
-                                    .then(function(){
+                                    //return self.createRedfishObm(encNode, device.ipAddress, 1)
+                                    //.then(function(){
```
Comment (MichaelRegert) [inline lib/jobs/dell-wsman-discovery.js]: Can we just delete these lines rather than checking in commented-out code?

File: lib/jobs/wsman-job.js [hunk 3]
```diff
@@ -51,169 +51,156 @@ function wsmanJobFactory(
...
-    
-    
-                  
-     return WsmanJob;
+     * Client Request API
+     * 
+     * 
+     */
+
+    WsmanJob.prototype.clientRequest = function (host, path, method, data) {
```
Comment (rahmanmuhamad) [inline lib/jobs/wsman-job.js]: How about removing this function and remove it with WsmanTool ? I know it was my bad to copy the function here

File: lib/jobs/dell-wsman-discovery.js [hunk 4]
```diff
@@ -726,9 +718,6 @@ function DellWsmanDiscoveryJobFactory(
                         case "SWITCH":
                             self.processSwitch(data);
                         break;
-                        case "VM":
```
Comment (phelpdh) [inline lib/jobs/dell-wsman-discovery.js]: Removed this case since it is not implemented.

File: lib/jobs/wsman-job.js [hunk 5]
```diff
@@ -2,24 +2,19 @@
...
-    'Assert',
     'Promise',
-    '_',
     'Services.Waterline',
-    'JobUtils.WsmanTool',       
-    'Constants',
+    'JobUtils.WsmanTool',
     'Services.Configuration',
     'Services.Encryption',
-    'HttpTool',
```
Comment (phelpdh) [inline lib/jobs/wsman-job.js]: Removed unused modules.

**Pull request merged**